finance/proposal-calls/2021-proposal/AperioMaintenance.md

```diff
@@ -0,0 +1,100 @@
+Infrastructure and core/coordinated package maintenance
+=======================================================
+
+### Project team
+
+This proposal is to fund the following people at
+[Aperio Software](https://aperiosoftware.com):
+
+* Thomas Robitaille
+* Stuart Mumford
+
+### Project Summary
+
+This project covers three main areas of work:
+
+* Maintenance of project infrastructure (such as pytest and sphinx plugins,
+  extension-helpers, continuous integration, wheel building, and so on). As
+  illustrated in
+  [astropy/astropy-project#118](https://github.com/astropy/astropy-project/issues/118),
+  the infrastructure for the project has grown significantly over the last few
+  years, and requires regular maintenance.
+* Maintenance and development of the
+  [astropy.timeseries](https://docs.astropy.org/en/stable/timeseries/) and
+  [astropy.visualization.wcsaxes](https://docs.astropy.org/en/stable/visualization/wcsaxes/)
+  sub-packages as well as the [reproject](http://reproject.readthedocs.io) and
+  [astropy-healpix](https://astropy-healpix.readthedocs.io/en/latest/)
+  coordinated packages.
+* Participation in project coordination and governance
+
+### Project / work
+
+#### Infrastructure work
```

Comment:
What about these?

* benchmarks
* stale bot (runs as cron separately from astropy-bot)
* big data storage solution (currently we rely on GitHub not blocking `astropy-data` repo access, which isn't sustainable if we want more data there)
* astropy-data updates from external people -- a lot of PRs to update `sites.json` but nothing is happening

Reply:
The first three I would be up for tackling. The last one is more an issue of organisation and responsibility; who is maintaining astropy-data and what's the policy on things that go in there.

Reply:
🤷

Reply:
It has been the source of quite some conflicts in the past. Until people having a strong opinion about what should not land in sites.json communicate it out in the open I think picking the task up is shooting oneself in the foot...

Reply:
Okay then, how about adding those first 3 points in your proposal? 😉

Reply:
Yes I agree about adding the first three!

Reply:
Have added